src/mame/includes/toaplan1.h

```diff
@@ -11,6 +11,7 @@
 
 #include "cpu/m68000/m68000.h"
 #include "cpu/tms32010/tms32010.h"
+#include "machine/toaplan_gxl.h"
 #include "sound/ymopl.h"
 #include "video/toaplan_scu.h"
```

Comment:
Headers from src/mame shouldn’t be intermixed with headers from src/devices.  Headers from src/mame should come first (most dependent to least dependent module).